fastplotlib/layouts/_figure.py

```diff
@@ -765,6 +769,14 @@ def remove_animation(self, func):
         if func in self._animate_funcs_post:
             self._animate_funcs_post.remove(func)
 
+    def clear_animations(self):
```

Comment:
takes a string kwarg to remove only "pre" or "post", default is `None` which clears both?